docs/manual/en/manual.txt

```diff
@@ -0,0 +1,300 @@
+// -*- mode: adoc; -*-
+
+:lang: en
+:toc-title: Table of Contents
+:last-update-label: Last update
+:warning-caption: Attention
```

Comment:
These heading titles were also overrides for non-English languages, I'm pretty sure they can go as well.